docs/android.md

```diff
@@ -109,6 +142,37 @@ those sections before configuring and building OpenDDS.
 PATH=$PATH:$TOOLCHAIN/bin make # Pass -j/--jobs with an appropriate value or this'll take a while...
 ```
 
+**Windows Users:** The command is similar with these exceptions:
+
+- `--host-tools` with the location of the OpenDDS host tools that were built
+  using Visual Studio must be passed to `configure`.
+
+- In addition to the Android toolchain, you will also need MinGW utilities in
+  your `%PATH` and the command prompt environment for Visual Studio. The
+  default installation location of the MinGW utilities is
+  `C:\MinGW\msys\1.0\bin`. The Visual Studio command prompt is the same as when
```

Comment:
Re-word this part (starting `The Visual Studio`), give the user explicit steps to run ("Start a new Visual Studio Command Prompt...").

Reply:
Yea, I had trouble wording this, making it explicit would be better.